sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -3211,8 +3213,7 @@ void simd_obj_impl<T, N, T1, SFINAE>::copy_to(
 
 template <typename T, int N, class T1, class SFINAE>
 template <typename AccessorT, typename Flags, int ChunkSize, typename>
-ESIMD_INLINE EnableIfAccessor<AccessorT, accessor_mode_cap::can_write,
-                              sycl::access::target::device, void>
+ESIMD_INLINE EnableIfAccessor<AccessorT, accessor_mode_cap::can_write, void>
```

Comment:
Same Q as above

Reply:
Good catch.
This case was supposed to be supported in the series of PRs implementing passing local_accessor to accessor-based memory functions.
We need a functions copy_from() & copy_to() accepting local accessor, but such variant needs to accept uint32_t even with __ESIMD_FORCE_STATELESS_MEM.

This function here should not accept local accessor because it accepts uint64_t in stateless mode.

Reply:
Additionally a separate variant of copy_to() copy_from() is needed for local accessor.